engine/runtime/src/test/java/org/enso/compiler/ParseStdLibTest.java

```diff
@@ -169,7 +169,8 @@ public void runBare() throws Throwable {
             "Data/Value_Type.enso",
             "Data/Vector.enso",
             "Network/HTTP/HTTP_Status_Code.enso",
-            "Internal/Base_Generator.enso"));
+            "Internal/Base_Generator.enso",
+            "System/File.enso"));
```

Comment:
If `System/File.enso` contains something that cannot be parsed by the old parser, then it is absolutely OK to excluded it.